lib/embulk/guess/jsonl.rb

```diff
@@ -20,8 +20,14 @@ def guess_lines(config, sample_lines)
 
         return {} if rows.size <= 3
 
-        columns = Embulk::Guess::SchemaGuess.from_hash_records(rows).map do |c|
-          column = {name: c.name, type: c.type}
+        if rows.empty?
+          raise "SchemaGuess Can't guess schema from no records"
```

Comment:
Is there any suitable Exception Class?

Reply:
I don't know

This is copied from https://github.com/embulk/embulk/blob/master/embulk-core/src/main/ruby/embulk/guess/schema_guess.rb#L19

Reply:
OK